### ISSUE TITLE
Test cubeviz base functionality with leaflet as image viz layer

### ISSUE THREAD

**Issue by SaOgaz** [spacetelescope/dat_pyinthesky#27]
From [functional spec](https://innerspace.stsci.edu/display/DSMO/Elaboration%3A+Cubeviz-in-browser+prototype?focusedCommentId=162048038#comment-162048038), do 1-3 using  ipyleaflet + a tile server a la Erik's demo + ipywidgets for interactive elements.

Stretch goal to achieve 4 (displaying in an ipywidgets text box but with cursor information from leaflet). Note that this may be very difficult, so it is "stretchier" than the others.

Wording of tasks (1-3) may be re-done in the near future, so keep that in mind.  Please feel free to post any and all implementation questions here in the issue!  This will be the best recorded forum to get any details worked out that might come up.

JIRA product board ticket: https://jira.stsci.edu/browse/DATJPP-17

**Comment by pllim** [comment]
I keep getting this warning from time to time, can this be safely ignored, @eteq ?

```
.../anaconda/envs/py37/lib/python3.7/site-packages/dask/config.py:161:
YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated,
as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
```

**Comment by eteq** [comment]
> can this be safely ignored?

Yes, I think so.  Although we should probably post an issue to dask about this if there isn't one already?

**Comment by pllim** [comment]
@eteq , good call. It appears to be addressed already in dask/dask#4597
